feat: add brief section

birkaany opened the pull request:
Brief section developed;
- BriefSection component has been created by using mobile-first approach
- Specified to this component classes added via BriefSection.module.css
- Added icons via Icon component's json file

Important note: "Title" component's style folder has been manipulated to style BriefSection's heading elements, this needs to be fixed/improved.

birkaany committed:
feat: add brief section

I've developed Brief section yet I manipulated Title.module.css to style custom heading styles. It needs to be checked.

eraygundogmus commented on components/Title/Title.module.css:
```diff
@@ -1,3 +1,23 @@
 .title {
-  @apply; /* TODO <-- add default title styles here */
+  @apply text-6xl; /* TODO <-- add default title styles here */
```
can you remove this comment?

birkaany replied:
Removed unnecessary comment lines

eraygundogmus commented on components/BriefSection/BriefSection.tsx:
```diff
@@ -0,0 +1,70 @@
+import Title from "../Title/Title";
+
+const BriefSection = () => {
+  const cardArray: any = [
```
can you create a sample.json and create this array on there? then you can import it here and i believe code will be cleaner on this component

birkaany replied:
I created a new json to storage data and exported into component

birkaany committed:
fix: add json to control card content data

- Removed unnecessary comment lines
- Created json file to storage card component's data

eraygundogmus reviewed (commented):
Thanks for your contribution
